OIPA/api/transaction/views.py

```diff
@@ -0,0 +1,29 @@
+from api.generics.views import DynamicListAPIView
+from api.generics.views import DynamicRetrieveAPIView
+from api.transaction.serializers import TransactionSerializer
+from iati.models import Transaction
+
+
+class TransactionDetail(DynamicRetrieveAPIView):
+    queryset = Transaction.objects.all()
+    serializer_class = TransactionSerializer
+
+
+class TransactionList(DynamicListAPIView):
+    queryset = Transaction.objects.all()
+    serializer_class = TransactionSerializer
+    fields = (
+        'id',
+        'url'
+        'activity_id',
```

Comment:
Any specific reason to show activity_id over here, and not a link or (partially)serialized activity?